graspy/utils/__init__.py

```diff
@@ -1,5 +1,41 @@
 # Copyright (c) Microsoft Corporation and contributors.
 # Licensed under the MIT License.
+from .utils import (
```

Comment:
I know this is tedious, but can you keep this in alphabetical order?  It can make it easier to quickly scan through later.